planning/behavior_path_dynamic_avoidance_module/include/behavior_path_dynamic_avoidance_module/scene.hpp

```diff
@@ -67,6 +67,12 @@ enum class PolygonGenerationMethod {
   OBJECT_PATH_BASE,
 };
 
+enum class ObjectBehaviorType {
+  NOT_TO_AVOID = 0,
+  LaneDriveObject,
+  FreeRunObject,
```

Comment:
FreeRunObject is hard to understand.

The best enum would be `ObjectType` which includes
- Vehicle
    - Technically, bicycles should be included, but in our case, not included.
- NotVehicle

, and the getter function's output is `std::optional<ObjectType>` to deal with objects not to avoid.

Or if you wanna add the ego's behavior to the enum, the enum would be `ObjectBehaviorType` which includes
- Ignore
- AvoidVehicleObject
- AvoidNotVehicleObject

I don't come up with a good name.

Reply:
Thank you.
I'll think these names again after the polygon generation algorithm is decided